README.md

```diff
@@ -1,77 +1,373 @@
 <h1 align="center"><img src="/img/vwc.gif" alt="Vets Who Code" width="100px" /> VETS WHO CODE</h1>
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Web Presence](#web-presence)&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;[Know Your Tool](#command-line)&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;[HTML & CSS](#html)&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;[JavaScript](#javascript)&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;[Capstone](#capstone)
-
 ---
 
 ## [Prework](https://vetswhocode.io)
 
-Welcome to the official Vets Who Code Prework.
-Here you'll find mandatory tutorials that need to be completed to be considered for the program.
-This can also be used as a great refresher for grads and newbies interested alike.
-While, we don't expect you to be experts, but we do expect you to be **familiar**.
-There is a lot of content to consume here, get used to the fire hose! 👨🏻‍🚒
+Welcome to the official Vets Who Code Prework! Here you'll find the tools, work, and accounts to prepare you for Vets Who Code's prior coursework.  We recommend you complete each section as they build upon one another. The prework should take a total of eight hours to complete.
```

Comment:
> prepare you for Vets Who Code's prior coursework

I'd change to `prepare you for Vets Who Code's coursework`.

> We recommend you complete each section as they build upon one another.

I'd change to `We recommend you complete each section as they build upon one another for the course.`

> The prework should take a total of eight hours to complete.
Should we have hour estimates marked next to each section?